app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/components/BrandTobAppBarScaffold.kt

```diff
@@ -28,10 +26,9 @@ fun BrandTobAppBarScaffold(
     floatingActionButton: @Composable () -> Unit = {},
     content: @Composable (PaddingValues) -> Unit,
 ) {
-    Scaffold(
-        topBar = { if (LocalNavType.current == NavigationSuiteType.NavigationBar) BrandTobAppBar() },
+    SmallWindowTopAppBarScaffold(
+        topBar = { BrandTobAppBar() },
```

Comment:
I renamed it using `tablet/phone` logic, because the names are simple and that's what we are using on a everyday basis when we are talking about this subject.

Reply:
If this needs to be renamed it needs to be done in the PR that will add the new naming convention so yeah, no need to bother with "small window" here